app/konnect/configure/runtime-manager/index.md

```diff
@@ -26,33 +34,44 @@ The Runtime Manager aims to simplify this process by providing a
 script to provision a {{site.base_gateway}} runtime in a Docker container,
 eliminating any confusion about initial configuration or setup.
 
-## Types of runtimes
+## Runtime groups
 
-### Kong Gateway
+Konnect manages runtime configuration in runtime groups. Each runtime group acts
+as a separate control plane and can manage runtime configurations independently
+of any other group. See [runtime groups](/konnect/configure/runtime-manager/runtime-groups)
+for more information.
 
-A {{site.base_gateway}} runtime acts as a data plane, which is a node
-serving traffic for the proxy. Data plane nodes are not directly connected
-to a database.
+## Types of runtimes
+
+Each runtime group connects to its own runtime instances. Every runtime instance
```

Comment:
I'd remove this line. It made me think that I could only run `docker` instances, or `vm` instances and not mix and match. The followup line explains enough